### DISCUSSION TITLE
Help With Direct Paths

### DISCUSSION THREAD

**Discussion by newoski** [croneter/PlexKodiConnect#2096] [Q&A] [answered]
Hi,

I've successfully setup Direct Paths via Emby in the past, but I've since migrated to Plex. I've got playback working via PlexKodiConnect add-on playback, but I can't get Direct Paths working despite hours of experimenting. I'm happy to use either Option 1 or Option 2. I don't know if it's appropriate to ask, but I'd really appreciate some guidance on where to go from here to get this setup.

1. I run Plex Server on my Unraid NAS and my Kodi client is an NVIDIA Shield Pro. 
2. My server is accessible via SMB and NFS. My preference would be to use NFS, but I'm happy to start with SMB
3. My server is accessible on my network via \\Tower and \\192.168.50.233 
4. I have triple-checked that my media is accessible to Kodi via the steps in your Wiki
5. My media is accessible via ONE root path: smb://Tower (ie smb://Tower/Movies/ and smb://Tower/TV/)
6. With my Plex docker in Unraid, I have mapped /mnt/user/ (the root directory of my server) as /Tower/ so the movies and tv shares are mounted in the Plex docker as /Tower/Movies and /Tower/TV

Is the above enough for you to provide a recommendation for which setup to use and what paths to use? I feel like I've tried every combination I can think of. 

Thanks so much!

**Comment by newoski** [comment]
@croneter Any chance you could assist?

**Comment by newoski** [comment, marked as answer]
For anyone with a similar issue, I finally solved this by NOT adding my Unraid NAS username/password during setup. Without those, everything works as expected for Direct Paths.

**Comment by DaywalkerNL** [comment]
Hello, i am new. ( sorry if english is not perfect, german is better ;-) )
I just installed PKC yesterday ( newest version ) for the first time. on a fresh kodi installation on a nvidia shield.
i got EXACT the SAME PROBLEM. my wish is to use the NFS File System ( KI says its the fastest and best way for skipping etc. in big mkv files with atmos ). but i didnt get it. cant set the right (direct) paths in PKC. or i am to stupid? ;-) - My Nas (Qnap) and NFS/SMB server are running. Pure Kodi can connect NFS without any problems. - Can someboy PLEASE HELP or explain the correct installation way for my situation? Thank you !
